renderer/components/Header.tsx

```diff
@@ -1,26 +1,30 @@
 import { featureFlags } from "@common/feature-flags";
 import React from "react";
 import Logo from "./icons/Logo";
+import { useAtomValue } from "jotai";
+import { translationAtom } from "@/atoms/translations-atom";
 
 export default function Header({ version }: { version: string }) {
+  const t = useAtomValue(translationAtom);
+
   return (
     <a
       href="https://github.com/upscayl/upscayl"
       target="_blank"
       className={`outline-none focus-visible:ring-2`}
       data-tooltip-id="tooltip"
-      data-tooltip-content="Star us on GitHub 😁"
+      data-tooltip-content={t("HEADER.GITHUB_BUTTON_TITLE")}
     >
       <div className="flex items-center gap-3 px-5 py-5">
         <Logo className="inline-block h-14 w-14" />
         <div className="flex flex-col justify-center">
           <h1 className="text-3xl font-bold">
-            Upscayl{" "}
+            {t("TITLE")}
             <span className="text-xs">
               {version} {featureFlags.APP_STORE_BUILD && "Mac"}
             </span>
```

Comment:
![image](https://github.com/user-attachments/assets/4bf8beca-d9bd-4e6a-8df9-b522c4b5dd2f)
This is how it looks like in my Chinese translation. Version number could use some padding.

Reply:
I'll check this out too..

Reply:
done.. pushed an update